Source/Applications/Application.cs

```diff
@@ -45,5 +45,7 @@ public static IApplicationBuilder WithName(ApplicationName name)
 
         /// <inheritdoc/>
         public IEnumerable<IApplicationLocationSegment> Prefixes { get; }
+
+        //TODO: Override GetHashCode?
```

Comment:
We generally don't want to have TODO / FIXME comments in our code, that gets hidden away and hard to know what todo - I've added an issue for this: https://github.com/dolittle/DotNET.Fundamentals/issues/25